### PR TITLE
webhooks: shorten PRs and issues' titles

### LINKED ISSUE #8: Shorten long text bodies
Some stuff that's displayed in messages can get quite long, should probably shorten some of the content

### PR DESCRIPTION
**Pull request by Max1Truc**
Fixes #8

### CODE REVIEW

**Comment by Max1Truc** [comment]
The shortener function is mixed in the event handling, maybe it should be somewhere else.

File: src/webhooks/github/events.rs [hunk 1]
```diff
@@ -21,6 +21,15 @@ pub use pull_request_review_comment::*;
 pub use push::*;
 pub use types::*;
 
+fn shorten_content(content: &str) -> String {
+    let max_length = 72;
+    if content.len() <= max_length {
+        content.to_owned()
+    } else {
+        content.split_at(max_length).0.to_owned() + "…"
```
Comment (alarsyo) [inline src/webhooks/github/events.rs:29]: This [will panic](https://doc.rust-lang.org/std/primitive.str.html#method.split_at) if 72 is in the middle of a unicode codepoint, if we're going to allocate a new `String` anyway to concatenate the `…`, might as well use something like `content.chars().take(max_length).collect()`

**Review by alarsyo — CHANGES_REQUESTED** [review]
Nitpick about Rust's Unicode handling

**Comment by alarsyo** [comment]
> The shortener function is mixed in the event handling, maybe it should be somewhere else.

How about something like `bot/utils.rs`? It's more related to bot/message functionality, although I've put some `Display` impls near their type declarations here.

Long term I'm picturing a `MessageBuilder` struct that'll provide helper functions for this and HTML formatting, but for now a `utils` mod is fine

**Review by alarsyo — CHANGES_REQUESTED** [review]
Some other place where text should be shortened (comment body):

https://github.com/prologin/prololo/blob/9e7a56a269c38be1ebc6e6c2b8d10796ef08b4fe/src/bot/github.rs#L136

**Review by alarsyo — APPROVED** [review]
LGTM!

**Pull request merged**